format/index.md

```diff
@@ -513,6 +513,39 @@ the following members:
   data represents a resource relationship.
 * Pagination links for the primary data (as described below).
 
+Value of each member of a `links` object can be either a string
+containing the link URL or a "link object", which can contain
+the following members:
+
+* `"href"` - the string containing the link URL or an array containing multiple strings containing the link URLs,
```

Comment:
I wouldn't bring this up if I didn't feel very strongly about it, since I know I'm beating a dead horse. But the more I think about it, the more convinced I am that having an array of target URIs for the same relation is something we're _going_ to want post 1.0—it's fundamental—such that I _really, really_ think we should include the array form in 1.0 to make the transition easier. Let me give two examples of the array form's value from _real issues that have already been raised_, that I felt I had to answer inadequately because the array form doesn't exist.

1. The first is the issue of [communicating hierarchical types](https://github.com/json-api/json-api/issues/571). That is, imagine an API with hierarchical collections for `products` > `appliances` > `cooking appliances` > `microwaves`, and a client requests `GET /products`. One of the resource objects returned from `/products` is a microwave, so its resource object has `"type": "microwaves"`, allowing the client to easily read any microwave-specific fields or apply chair-specific presentation. But, for the client to check whether the resource is also an appliance or a cooking appliance (e.g. to offer the user filters in the UI), I [suggested](https://github.com/json-api/json-api/issues/571#issuecomment-97376061) relying on information in `meta`. However, since hierarchical types are relatively common, it'd be better if this could be exposed to generic clients. And a `collection` link that could hold an array of ([optionally named](https://github.com/json-api/json-api/issues/622#user-content-named-collections)) targets would be a _perfect_, standards compliant way to state those `is-a` relationships.

2. The second example is this request for [linking to alternate versions of a resource in different languages](https://github.com/json-api/json-api/issues/613). I suggested making an `alternates` relationship but, of course, that's just a poor-man's version of an `alternate` link relation with an array of targets!

If those aren't convincing, then I'll drop it forever. But I just wanted to make one last stab at this with some real world, already present issues.